api/api/views/media_views.py

```diff
@@ -88,17 +88,21 @@ def list(self, request, *_, **__):
         if qa:
             logger.info("Using QA index for media.")
             search_index = self.qa_index
+            exact_index = False
```

Comment:
I'm not familiar with what the `qa_index` is for so I could be wrong, but I don't think we have a `-filtered` version of this either.

Reply:
The `qa` index is mostly defunct, it has like 4 handwritten entries. We should probably deprecate it entirely and just remove all references pointing to it.

Reply:
Sounds reasonable to me!